### PR TITLE
LCSD-4273 - SEP service stub

### PR DESCRIPTION
**Pull request by GeorgeWalker**

### CODE REVIEW

File: sep-service/ViewModels/Address.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace sep_service.ViewModels
```
Comment (asanchezr) [inline sep-service/ViewModels/Address.cs:6]: Are these models auto-generated? 
My only comment here would be to try to have the namespace conform better to C# naming guidelines: 
* `sep_service.ViewModels` --> `SepService.ViewModels`

Thoughts?

File: sep-service/ViewModels/Sol.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
+using System;
+using sep_service.ViewModels;
+
+namespace sep_service
+{
+    public class Sol
```
Comment (asanchezr) [inline sep-service/ViewModels/Sol.cs:6]: If possible would love to see here what "Sol" means? I can infer the other view models but this one cannot...

File: sep-service/ViewModels/Address.cs (hunk 1 above)
Comment (GeorgeWalker) [reply sep-service/ViewModels/Address.cs:6]: It's auto generated, but I can adjust it.

File: sep-service/ViewModels/Sol.cs (hunk 2 above)
Comment (GeorgeWalker) [reply sep-service/ViewModels/Sol.cs:6]: It's from the model given to use by the other vendor.  I am not aware of what it means

**Review by asanchezr — APPROVED** [review]

**Pull request merged**